ASP_tests.py

```diff
@@ -273,43 +277,70 @@ def test_make_regular_grid():
+                                      size=25, spacing=3.0)
+    assert np.allclose(ra_grid2, test_ra, atol=1e-9), "RA vals do not match for Snappl WCS"
+    assert np.allclose(dec_grid2, test_dec, atol=1e-9), "Dec vals do not match for Snappl WCS"
+
 
 
 def test_make_adaptive_grid():
     wcs = np.load('./tests/testdata/wcs_dict.npz', allow_pickle=True)
-    wcs = dict(wcs)
-    ra_center = wcs['CRVAL1']
-    dec_center = wcs['CRVAL2']
+    wcs_dict = dict(wcs)
+    ra_center = wcs_dict['CRVAL1']
+    dec_center = wcs_dict['CRVAL2']
     for key in wcs.keys():
-        wcs[key] = wcs[key].item()
-    wcs = galsim.wcs.readFromFitsHeader(wcs)[0]
+        wcs_dict[key] = wcs_dict[key].item()
+    wcs = galsim.wcs.readFromFitsHeader(wcs_dict)[0]
     compare_images = np.load('tests/testdata/images.npy')
     image = compare_images[:11**2].reshape(11, 11)
     ra_grid, dec_grid = make_adaptive_grid(ra_center, dec_center, wcs,
                                            image=image, percentiles=[99])
     test_ra = [7.67356034, 7.67359491, 7.67362949, 7.67366407, 7.67369864,]
     test_dec = [-44.26425446, -44.26423765, -44.26422084, -44.26420403, -44.26418721]
-    assert np.allclose(ra_grid[:5], test_ra, atol=1e-7), "RA vals do not match"
-    assert np.allclose(dec_grid[:5], test_dec, atol=1e-7), "Dec vals do not match"
+    assert np.allclose(ra_grid[:5], test_ra, atol=1e-9), "RA vals do not match using Galsim WCS"
+    assert np.allclose(dec_grid[:5], test_dec, atol=1e-9), "Dec vals do not match using Galsim WCS"
+
+    # Astropy / Snappl WCS
+    wcs = WCS(wcs_dict)
+    ra_grid2, dec_grid2 = make_adaptive_grid(ra_center, dec_center, wcs,
+                                             image=image, percentiles=[99])
+    assert np.allclose(ra_grid2[:5], test_ra, atol=1e-9), "RA vals do not match using snappl wcs"
+    assert np.allclose(dec_grid2[:5], test_dec, atol=1e-9), "Dec vals do not match using snappl wcs"
 
 
 def test_make_contour_grid():
     wcs = np.load('./tests/testdata/wcs_dict.npz', allow_pickle=True)
-    wcs = dict(wcs)
-    for key in wcs.keys():
-        wcs[key] = wcs[key].item()
-    wcs = galsim.wcs.readFromFitsHeader(wcs)[0]
+    wcs_dict = dict(wcs)
+    for key in wcs_dict.keys():
+        wcs_dict[key] = wcs_dict[key].item()
+
+    # Galsim WCS
+    wcs = galsim.wcs.readFromFitsHeader(wcs_dict)[0]
     compare_images = np.load('tests/testdata/images.npy')
     image = compare_images[:11**2].reshape(11, 11)
     ra_grid, dec_grid = make_contour_grid(image, wcs)
     test_ra = [7.67356034, 7.67359491, 7.67362949, 7.67366407]
     test_dec = [-44.26425446, -44.26423765, -44.26422084, -44.26420403]
-    msg = "RA vals do not match"
-    assert np.allclose(ra_grid[:4], test_ra, atol=1e-7), msg
-    msg = "Dec vals do not match"
-    assert np.allclose(dec_grid[:4], test_dec, atol=1e-7), msg
+    Lager.debug('ra_grid: %s', ra_grid[:10])
+    Lager.debug(ra_grid[:4] - test_ra)
+    msg = "RA vals do not match to 1e-9"
```

Comment:
Define variable that defines 1e-9 to make sure you're consistent in the `msg` and the actual test.

But also, I don't think you have to do all of these custom messages when using `pytest` and `numpy.testing`.  I think it will automatically generate these things.